thesis-de/content-de.tex

```diff
@@ -1175,10 +1175,30 @@ \subsubsection{Durchführung der Evaluierung}
 
 \paragraph{Evaluierungsbericht}
 \label{paragraph:hadoop_hive_impala_evaluierung}
-Das ganze ist kacke weil zu viel Some more Text here
+Während der Evaluierungsphase wurde ein vollständiger Hadoop Core als Prototyp
+auf zwei Testservern installiert und in den Grundlagen konfiguriert. Während
+dieser Arbeit ist dem Projektteam bereits klar geworden, dass nur für die
+Konfiguration des ersten Systemstarts, also dem Verbindungsaufbau, sowie den
+allgemeinen Funktionen eines Hadoop Core Systems zu viel Zeit
+benötigt wird. Desweiteren wurde nach dem testen des Systems die
+Konfigurationsmöglichkeiten im Bereich Optimierung der Performance und
+Geschwindigkeit studiert. Dabei wurde klar, dass durch die Größe des
+Systems die gesamte Projektzeit benötigt wird, um eine optimale und
+vollständige Konfiguration und Systemstruktur zu erarbeiten. Aufgrund
+dessen wurde die Verwendung eines Hadoop Systems in diesem Projekt sehr
+schnell ausgeschlossen. Hauptgründe dabei waren zum einem die vielen Werkzeuge
+und Methoden die das Hadoop Ecosystem bereitstellen konnte und zum anderen
+die Masse an Dokumentationen und dem damit verbundenen Konfigurationsaufwand.
+Hinzu kommt noch die darauf folgende fachliche Definition von Datenstrukturen,
+welche durch die breite Auswahl an Methodiken auf einem Hadoop System ebenfalls
+wieder viel Zeit beansprucht hätte, da zunächst aus einem der verfügbaren
+Softwareelementen aus dem Hadoop Ecosystem evaluiert und ausgewählt werden
+müsste. Zusammengefasst wäre das System den technischen Anforderungen an dem
+Projekt gewachsen, jedoch wäre der Einsatz mit einem zu großen Aufwand
+verbunden, welchen wir in dem vorgegebenen Projektzeitraum nicht erbringen
```

Comment:
falsche erzählperspektive. '... welchen das Projektteam in dem vorgegeben Zeitraum nicht erbingen kann'